### PR TITLE
[JENKINS-66708] Modernize config-file-provider icons

### PR DESCRIPTION
**Pull request by bzzitsme**
- [`x`] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [`x`] Ensure that the pull request title represents the desired changelog entry
- [`x`] Please describe what you did

PR for https://issues.jenkins.io/browse/JENKINS-66708
C-F-G icon was changed to 
cfg-logo

If you have any suggestions or questions regarding PR, I will be happy to hear/answer them.

We could also use this icon instead of one that is in the PR:
file-copy

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
         <changelist>-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
         <maven.javadoc.skip>true</maven.javadoc.skip>
-        <jenkins.version>2.303.3</jenkins.version>
+        <jenkins.version>2.346</jenkins.version>
```
Comment (alecharp) [inline pom.xml:34]: I appreciate the upgrade, but I'd prefer to keep a LTS here.

```suggestion
        <jenkins.version>2.346.1</jenkins.version>
```

**Review by alecharp — CHANGES_REQUESTED** [review]
This looks great. I tested it locally and it was working fine. 
Could you just update the `jenkins.version` to keep referencing to an LTS? Thanks.

File: pom.xml (hunk 1 above)
Comment (alecharp) [reply pom.xml]: Thanks.

**Review by jmMeessen — COMMENTED** [review]
LGTM

**Comment by jmMeessen** [comment]
Thank you for the pull request @bzzitsme. Merged it and cutting a release

**Comment by NotMyFault** [comment]
The proposed symbol doesn't honor the API and is not themable.

**Comment by bzzitsme** [comment]
> is not themable

Do you mean dark theme plugin?

**Comment by NotMyFault** [comment]
> > is not themable
> 
> Do you mean dark theme plugin?

In any theme plugin. The API looks for `currentColor` values in common attributes like stroke, fill, etc. used to design SVGs.
If none is present, Jenkins can't apply a theme.

**Comment by NotMyFault** [comment]
I just noticed this PR, because I got a mail sent by my Jira issue. But that's what I had in mind when reviewing the core PR.
I'm not sure if that is stated somewhere, if not, I'll write something up.

**Pull request merged**